kotlin-inject-compiler/kapt/src/main/kotlin/me/tatarka/inject/compiler/kapt/Util.kt

```diff
@@ -301,6 +301,18 @@ fun KmType.eqvHashCode(collector: HashCollector = HashCollector()): Int {
     }
 }
 
+/**
+ * Returns the [KmType] without type alias info.
+ */
+fun KmType.resolve(): KmType = KmType(flags).also {
```

Comment:
Maybe use `apply` here? At first glance this doesn't seem to be performing any side effects so I would avoid using also here

Reply:
I did originally, but imo
```
classifier = classifier
```
is even more confusing

Reply:
Makes sense. Probably worth adding a comment about. LGTM 👍